docs/quickstart/index.md

```diff
@@ -0,0 +1,152 @@
+
+### Connect database
+
+Once PMM is set up, choose the database that you want it to monitor:
+
+=== "MySQL"
+
+    Follow the instructions below to connect a Self-hosted MySQL database. Alternatively, you can connect an [AWS RDS](../setting-up/client/aws.md), [Azure MySQL](../setting-up/client/azure.md) or a [Google Cloud MySQL ](../setting-up/client/google.md) database.
+    { .power-number }
+
+    1. Create database account for PMM using the following command example. This creates a database user with name `pmm``, password `pass``, and the necessary permissions:
+
+        ```sql
+        CREATE USER 'pmm'@'127.0.0.1' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 10;
+        GRANT SELECT, PROCESS, REPLICATION CLIENT, RELOAD, BACKUP_ADMIN ON *.* TO 'pmm'@'127.0.0.1';
+        ```
+
+    2. To optimize server-side resources, install PMM Client via Package Manager on the database node:
+         
+        1. Install Percona Release Tool:
+
+            ```sh
+            wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+            dpkg -i percona-release_latest.generic_all.deb
+            ```
+        2. Install the PMM Client package:
+            
+            ```sh
+            apt update
+            apt install -y pmm2-client
+            ```
+
+        3. [Register database node with PMM](../setting-up/client/index.html#register):
+                
+            ```sh
+            pmm-admin config --server-insecure-tls --server-url=https://admin:admin@X.X.X.X:443
+            ```
+
+        4. Add the MySQL database using Performance schema:  
+
+            ```sh 
+            pmm-admin add mysql --query-source=perfschema --username=pmm --password=pass MYSQL_NODE
+            ```
+
+        Alternatively, you can install PMM Client as a Docker container or as a binary package. See [alternative PMM Client installation options](../setting-up/client/index.html#binary-package).
+
+        If you don't have access to the database node, [install PMM Client from the User Interface](../setting-up/client/mysql.html#with-the-user-interface) instead. 
+
+=== "PostgreSQL"
+
+    **Prerequisites**
+     
+    Before adding a PostgreSQL database for monitoring, [create a database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#create-a-database-account-for-pmm)
+    
+    **Add a PostgreSQL database instance**
+    {.power-number}
+
+    To add a PostgreSQL database instance for monitoring, do the following:
+
+    1. From the PMM UI, go to **Configuration > PMM Inventory > Add Instance** and choose **PostgreSQL**.
+
+    2. Enter your database credentials on the resulting page.
+
+    3. (Optional) Enter the information on the **Labels** or **Additional Options** section. 
+
+    4. Click **Add Service** at the bottom.
+
+    For detailed information, see [Adding a PostgreSQL database](../setting-up/client/postgresql.md).
```

Comment:
why do we recommend to use UI to add PostgreSQL and MongoDB, but CLI for MySQL

Reply:
Removed this file from this branch as it is not related to this PR.